internal/kafka/command_topic_produce.go

```diff
@@ -205,22 +206,22 @@ func GetProduceMessage(cmd *cobra.Command, keyMetaInfo, valueMetaInfo []byte, to
 }
 
 func serializeMessage(keyMetaInfo, valueMetaInfo []byte, data, delimiter string, parseKey bool, keySerializer, valueSerializer serdes.SerializationProvider) ([]byte, []byte, error) {
-	var serializedKey, val string
+	var serializedKey []byte
+	var val string
```

Comment:
This should let you delete the `else` branch
```suggestion
	val := data
```